### PR TITLE
general dashboard cleanup last step

### PR DESCRIPTION
**Pull request by andi-halim**
[//]: # "These comments meant for your reference, they are invisible and don't need to be deleted"

## What's new in this PR?

Removed files for general dashboard announcements. Expanded white box for general dashboard and changed css. got rid of people power card component with photo. 

### How to Review

[//]: # 'The order in which to review files and 
what to expect when testing locally'
[//]: # '####### YOUR ANSWER BELOW ###########'
[//]: # '############## END ##################'

### Related PRs

[//]: # "Optional - related PRs you're waiting on
/ PRs that will conflict, etc"
[//]: # '####### YOUR ANSWER BELOW ###########'
[//]: # '############## END ##################'

### Next steps

[//]: # "What doesn't work yet, what's NOT in this 
PR/has to be done "
[//]: # '####### YOUR ANSWER BELOW ###########'
[//]: # '############## END ##################'

### Screenshots

Before: 
image

AFTER: 
image

### Design Status

[//]: # 'If this is a frontend PR, what is the expected 
status of the UI in this PR (lo, mid, high- fi?'
[//]: # '####### LOFI/MIDFI/HIFI ###########'
[//]: # '############## END ##################'

CC: @dfangshuo

### CODE REVIEW

**Review by phoebeli23 — APPROVED** [review]

**Review by aliceyezhao — APPROVED** [review]

File: src/styles/GeneralOwnerDashboard.css [hunk 1]
```diff
@@ -55,12 +53,13 @@
...
+.dashboard-box {
+  border-radius: 5px;
+  background-color: white;
+  padding: 2rem;
   width: 100%;
+  height: 70vh;
   display: flex;
-  justify-content: center;
+  flex-direction: column;
+  margin-right: 10rem;
```
Comment (niwong) [inline src/styles/GeneralOwnerDashboard.css]: --

File: src/styles/GeneralOwnerDashboard.css [hunk 2]
```diff
@@ -10,15 +10,13 @@
   background: white;
 }
 
-.dashboard .dash-solar-details-cont,
-.dash-announcements-cont {
+.dashboard {
   margin-top: 55px;
```
Comment (niwong) [inline src/styles/GeneralOwnerDashboard.css]: Replace all margin tags from this class and replace with `margin: 60px`

Comment (niwong) [reply src/styles/GeneralOwnerDashboard.css]: Also remove `width` from `.dashboard`. This should make the white box a bit nicer.

File: src/styles/GeneralOwnerDashboard.css (hunk 1 above)
Comment (niwong) [reply src/styles/GeneralOwnerDashboard.css]: Remove `margin-right: 10rem`

**Review by niwong — CHANGES_REQUESTED** [review]
Go ahead and make those changes that I commented and feel free to merge in when finished! Nice job, Andy.

**Review by niwong — APPROVED** [review]
Looks great!

**Pull request merged**